tests/settings/ConfigurationGatheringTest.py

```diff
@@ -101,13 +101,50 @@ def test_default_coafile_parsing(self):
 
         Constants.system_coafile = tmp
 
+    def test_capture_warnings_for_default_coafile(self):
+        """
+        This test case is temporary.
+        """
+        tmp = Constants.system_coafile
+
+        Constants.system_coafile = os.path.abspath(os.path.join(
+            os.path.dirname(os.path.realpath(__file__)),
+            'section_manager_test_files',
+            'default_coafile'))
+
+        logger = logging.getLogger(__name__)
+        with self.assertLogs(logger, 'WARNING') as log:
+            self.assertEqual(log.output,
+                             ["WARNING: root: Use of file named\
```

Comment:
Please replace `"` with `'`

Reply:
and please don't use backslash continuation^^